src/types/index.ts

```diff
@@ -366,7 +366,7 @@ export type MultiClaimCondition = ConditionBase & {
 
 export type IdentityCondition = ConditionBase & {
   type: ConditionType.IsIdentity;
-  identity: Identity;
+  identity: Identity | string;
```

Comment:
This would affect the type when you're returning conditions to the user. The idea was to only change the type that is used for input